locations/dict_parser.py

```diff
@@ -147,12 +116,77 @@ def parse(obj) -> GeojsonPointItem:
     @staticmethod
     def get_first_key(obj, keys):
         for key in keys:
-            if obj.get(key):
-                return obj[key]
-            elif obj.get(key.lower()):
-                return obj[key.lower()]
-            elif obj.get(key.upper()):
-                return obj[key.upper()]
+            variations = DictParser.get_variations(key)
+            for variation in variations:
+                if obj.get(variation):
+                    return obj[variation]
+
+    @staticmethod
+    def get_variations(key):
+        results = [key]
```

Comment:
Can you use a `set()` instead of an array? That'll save all the `if not x in results`.

Reply:
I didn't know that was a thing, thank you.